vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
I think we're talking about slightly different things. I was talking about what would happen if there was a precinct that linked two Electoral Districts *actually* of the same type; that's not a problem. What @jktomer is talking about is a precinct linking to two Electoral Districts which are *not actually* of the same type, combined with multiple sources of ballot information, combined with the current internal logic of our pipeline for de-duplicating from those two sources.